bug/bug_test.go

```diff
@@ -99,6 +100,12 @@ func TestBugCommitLoad(t *testing.T) {
 	bug3, err := ReadLocalBug(repo, bug1.Id())
 	assert.NoError(t, err)
 	equivalentBug(t, bug1, bug3)
+
+	err = RemoveLocalBug(repo, bug1.Id())
+	assert.NoError(t, err)
+
+	streamedBugs := ReadAllLocalBugs(repo)
+	assert.Equal(t, 0, len(streamedBugs))
```

Comment:
It's subtle, but there is a race here. `streamedBugs` is a channel. `len(streamedBugs)` return the number of element **at the moment** in the channel, but `ReadAllLocalBugs()` might not have started pushing values there yet.

To count the number or bugs, you need to really read the channel and count the values.